sample/example/example.ta.wordlist/source/example.ta.wordlist.model.kps

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>12.0.53.0</KeymanDeveloperVersion>
+    <FileVersion>12.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">Example (Tamil) Template Wordlist Model</Name>
+    <Version>1.0</Version>
+    <Copyright URL="">© 2019 SIL International</Copyright>
+    <Author URL="">Darcy Wong</Author>
```

Comment:
Please add an email (even a dummy one like `mailto:nobody@example.com`) here.

Reply:
ooh, thanks for pointing that out! It builds now.